.github/workflows/deploy.yml

```diff
@@ -53,4 +55,4 @@ jobs:
           heroku_email: ${{ secrets.HEROKU_ACCOUNT_EMAIL }}
           usedocker: true
           docker_build_args: |
-            DATABASE_URL
+            DATABASE_URL=${{ secrets.DATABASE_URL_PRODUCTION }}
```

Comment:
The set just before `name: Set DATABASE_URL` is doing that automatically.
- when deploying any branch that is NOT main, it will use `secrets.DATABASE_URL_STAGING`
- when deploying the `main` branch, it will use `secrets.DATABASE_URL_PRODUCTION`

So it seems we don't need to change that, but only to add the needed secrets in GitHub 🤔